packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -553,7 +521,7 @@ export interface SdkHttpResponse extends SdkServiceResponse {
   kind: "http";
   contentTypes?: string[];
   defaultContentType?: string;
-  description?: string;
+  doc?: string;
```

Comment:
I made this breaking change so it's .doc everywhere, @tadelesh are you ok with that?

Reply:
the reason i keep it `description` is it is not a direct type doc from typespec, but a property named description from http response in http lib. but i'm open for the change. @haolingdong-msft could you confirm if this change has any impact for java (you introduced it at first).

Reply:
How does http lib name it? `description` or `doc`? I would prefer to be consistent with http lib's name.

Reply:
description

Reply:
i did `doc` just for consistency in `tcgc`, but yeah I can keep it `description`. Not sure why it's different honestly between the libraries

Reply:
let me keep it `description` for consistency with `http` lib for now, we can always change later if tcgc users get too confused